packages/devtools-launchpad/src/components/shared/menu.js

```diff
@@ -71,11 +71,13 @@ function showMenu(evt, items) {
   }
 
   let menu = new Menu();
-  items.filter((item) => item.visible).forEach((item) => {
-    let menuItem = new MenuItem(item);
-    menuItem.submenu = createSubMenu(item.submenu);
-    menu.append(menuItem);
-  });
+  items
+    .filter((item) => item.visible === undefined || item.visible === true)
```

Comment:
subMenu.items.filter fixed!

I know devtools-core is very unstable and sometime the new PR broke usability. But I think we haven't setup unit test or integration test framework in devtools-core yet. We can add a test suite for menu once testing framework is settle down.